project.clj

```diff
@@ -10,17 +10,22 @@
             [test2junit "1.4.4"]]
   :dependencies [[org.clojure/clojure "1.11.4"]
                  [org.clojure/tools.logging "1.3.0"]
-                 [org.irods.jargon/jargon-core "4.2.2.1-RELEASE"
+                 [org.irods.jargon/jargon-core "4.3.7.0-RELEASE"
                   :exclusions [[org.jglobus/JGlobus-Core]
                                [org.slf4j/slf4j-api]
                                [org.slf4j/slf4j-log4j12]]]
-                 [org.irods.jargon/jargon-data-utils "4.2.2.1-RELEASE"
+                 [org.irods.jargon/jargon-data-utils "4.3.7.0-RELEASE"
                   :exclusions [[org.slf4j/slf4j-api]
                                [org.slf4j/slf4j-log4j12]]]
-                 [org.irods.jargon/jargon-ticket "4.2.2.1-RELEASE"
+                 [org.irods.jargon/jargon-ticket "4.3.7.0-RELEASE"
                   :exclusions [[org.slf4j/slf4j-api]
                                [org.slf4j/slf4j-log4j12]]]
-                 [cheshire "5.13.0"]
+                 [cheshire "5.13.0"
+                  :exclusions [[com.fasterxml.jackson.core/jackson-databind]]]
+                 [com.fasterxml.jackson.dataformat/jackson-dataformat-cbor "2.14.1"
+                  :exclusions [[com.fasterxml.jackson.core/jackson-databind]]]
+                 [com.fasterxml.jackson.dataformat/jackson-dataformat-smile "2.14.1"
+                  :exclusions [[com.fasterxml.jackson.core/jackson-databind]]]
```

Comment:
There was a Jackson dependency conflict that was preventing the library from compiling correctly. The latest version of Jargon uses Jackson 2.14.1, so we're forcing this library to use the same version of it.